testinfra/vars/app-staging.yml

```diff
@@ -22,8 +22,8 @@ apache_source_log: /var/log/apache2/source-error.log
 apache_allow_from: all
 
 dns_server: "8.8.8.8"
-mon_ip: 10.0.1.3
-app_ip: 10.0.1.2
+mon_ip: 172.16.1.203
+app_ip: 172.16.1.69
```

Comment:
I spy a regression here... these changes should not have been committed to version control.